joplin/base/wagtail_hooks.py

```diff
@@ -163,7 +164,14 @@ def joplin_page_listing_buttons(page, page_perms, is_parent=False):
         )
 
     # make the author notes icon appear if latest revision has notes
-    latest_revision_as_page = page.get_latest_revision_as_page()
+    # TODO this is suddenly causing a permissions error which is breaking on some pages
```

Comment:
Forgive me if you posted the error somewhere already, but what was the error out of curiousity

Reply:
It eventually ended up being this:
https://github.com/cityofaustin/techstack/issues/3842